lib/ExpensiMark.js

```diff
@@ -142,7 +128,7 @@ export default class ExpensiMark {
 
                 process: (textToProcess, replacement) => {
                     const regex = new RegExp(
-                        `(?![^<]*>|[^<>]*<\\/)([_*~]*?)${MARKDOWN_URL_REGEX}\\1(?!((?:(?!<a).)+)?<\\/a>|[^<]*(<\\/pre>|<\\/code>))`,
+                        `(?![^<]*>|[^<>]*<\\/)([_*~]*?)${MARKDOWN_URL_REGEX}\\1(?!(@(?:[a-z\\d-]+\\.)+[a-z]{2,})|((?:(?!<a).)+)?<\\/a>|[^<]*(<\\/pre>|<\\/code>))`,
```

Comment:
Wouldn't it be better to add the negative lookahead directly in `URL_WEBSITE_REGEX` so that it applies before the `URL_PARAM_REGEX` (like was proposed [here](https://github.com/Expensify/App/issues/16762#issuecomment-1500807823))?